model/client4.go

```diff
@@ -5605,3 +5605,13 @@ func (c *Client4) UploadData(uploadId string, data io.Reader) (*FileInfo, *Respo
 	defer closeBody(r)
 	return FileInfoFromJson(r.Body), BuildResponse(r)
 }
+
+// InvalidateAllEmailInvites invalidates all email invites.
+func (c *Client4) InvalidateAllEmailInvites() (bool, *Response) {
+	r, err := c.DoApiDelete(c.GetTeamsRoute() + "/invites/email")
+	if err != nil {
+		return false, BuildErrorResponse(r, err)
+	}
+	defer closeBody(r)
+	return CheckStatusOK(r), BuildResponse(r)
+}
```

Comment:
I believe this existed already. Did you miss it or are we duplicating it on purpose?

https://github.com/mattermost/mattermost-server/blob/96d3ddd4f10662cca856c2c197629f58527d16f9/model/client4.go#L2215-L2223

Reply:
I missed it, thanks for catching this, `cmd+shift+f` failed me 😛